### ISSUE TITLE
Request: Bridge Builder Cart

### ISSUE THREAD

**Issue by Lightwave413** [Railcraft/Railcraft#380]
Upgrade to the Track Layer Cart:
First places a block, then lays track on top.
Extremely useful for creating elevated railways.

**Comment by choncho** [comment]
Yes this will be very nice especally for H.S tracks

**Comment by liach** [comment]
But now the job can be done with force track and track relayer.

**Comment by Lightwave413** [comment]
true, but the emitter has limited range.  Using that setup requires player interaction every 64 blocks and an established energy transmission infrastructure spanning the length of the bridge to be constructed.
